zerver/tests/test_subs.py

```diff
@@ -1655,6 +1655,26 @@ def test_subscriber_private_stream(self):
                                          invite_only=True)
         self.make_successful_subscriber_request(stream_name)
 
+    def test_json_get_subscribers(self):
```

Comment:
Can you make the commit message for this commit be `Add test_json_get_subscribers`?